README.md

```diff
@@ -1,100 +1,94 @@
+- DeOldify: ![preview](assets/examples/de-oldify.jpg)
 
-To distribute your Library please refer to the [Library Guidelines](https://github.com/processing/processing/wiki/Library-Guidelines).
+- Face-Landmarks: ![preview](assets/examples/face-landmarks.jpg)
 
-## Source code
+- Face-Recognition
 
-If you want to share your Library's source code, we recommend using an online repository available for free at [GitHub](https://github.com/).
+- Fast-Style-Transfer: ![preview](assets/examples/fast-style-transfer.jpg)
 
-## <a name='AddingJARs'/>Adding core.jar and other .jar files to your classpath</a>
+- GPT-2
 
-The `core.jar` file contains the core classes of Processing and has to be part of your classpath when building a Library. On Windows and Linux, this file is located in the Processing distribution folder inside a folder named `lib`. On Mac OS X, right-click the Processing.app and use "Show Package Contents" to see the guts. The `core.jar` file is inside Contents → Resources → Java. For further information about the classes in `core.jar`, you can see the source [here](http://code.google.com/p/processing/source/browse/trunk/processing#processing/core) and the developer documentation [here](http://processing.googlecode.com/svn/trunk/processing/build/javadoc/core/index.html).
+- im2txt
 
-If you created a `libs` folder as described above, put the libraries you need to add to your classpath in there. In the "Properties" of your Java project, navigate to Java Build Path → Libraries, and click "Add External JARs...". Select the `.jar` files from the `libs` folder that are required for compiling your project. Adjust the `build.xml` file accordingly.
+- Image-Inpainting-GMCNN
 
-The `libs` folder is recommended but not a requirement, nevertheless you need to specify where your `.jar` files are located in your system in order to add them to the classpath.
+- Image-Super-Resolution: ![preview](assets/examples/image-super-resolution.jpg)
 
-In case a Library depends on system libraries, put these dependencies next to the `.jar` file. For example, Processing's `opengl.jar` Library depends on JOGL hence the DLLs (for Windows) or jnilibs (for OS X) have to be located next to the `opengl.jar` file.
+- MaskRCNN
+
+- Model3DDFA
+
+- OpenPifPaf-Pose
+
+- Photo-Sketch: ![preview](assets/examples/photo-sketch.jpg)
+
+- PoseNet: ![preview](assets/examples/posenet.jpg)
+
+- SPADE-COCO
+
+- SPADE-Landmarks
+
+- StyleGAN: ![preview](assets/examples/style-gan.jpg)
+
+- VisualImportance
+
+- YOLACT
+  
 
-## What is the difference between JDK and JRE?
+## Dependencies
```

Comment:
Perhaps a "Getting Started" section would make sense but I think it's better if we merge this PR first and then add a few resources for people to get started in another PR. Let's keep it like this for now.